wiki/Ranking_Criteria/zh.md

```diff
@@ -1,22 +1,19 @@
----
-outdated: true
-outdated_since: 2da3d3ec26261552cc7ae67298d375b939addbf3
----
-
 # Ranking Criteria
 
 *查看每个游戏模式具体的标准，请参阅 [osu!](/wiki/rc_osu!)，[osu!taiko](/wiki/rc_osu!taiko)，[osu!catch](/wiki/rc_osu!catch) 和 [osu!mania](/wiki/rc_osu!mania)。*
 
-这里会列出[官方论坛](https://osu.ppy.sh/community/forums/87)中的新增内容。按照惯例，任何正在讨论的规定都将列在此处，一旦讨论达成一致，就会更新它们。
+**ranking criteria** 列出了让[谱面](/wiki/Beatmaps)通过[谱面ranking程序](/wiki/Beatmap_ranking_procedure)必须遵守的[规则和准则](#general-terms)
+
+这里会列出 [Ranking Criteria 论坛](https://osu.ppy.sh/community/forums/87)中的新增内容。参照 *[如何建议更改Ranking准则](https://osu.ppy.sh/community/forums/topics/720532)* 来提供贡献。 注意在仅改变文档的语言，语法或者组织而不改变其规则和准则的意思的情况下无需发帖至论坛。
 
-**注意[行为准则](/wiki/Code_of_Conduct)，[8 分音符 Timing](/wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures)和[歌曲内容规定](/wiki/Ranking_Criteria/Song_Content_Rules)是 Ranking Criteria 的一部分，适用于 osu! 中所有游戏模式。**
+**注意[行为准则](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)，[8 分音符 Timing](/wiki/Ranking_Criteria/Timing_Songs_With_8-Signatures)和[歌曲内容规定](/wiki/Rules/Song_Content_Rules)是 Ranking Criteria 的一部分，适用于 osu! 中所有游戏模式。**
```

Comment:
光“行为准则”几个字没有涵盖到Modding and Mapping的意思，略有些不妥。
建议翻译为为“摸图和作图行为行为准则”。

EN: The related article title has since changed from just "Code of Conduct" to "Code of
 Conduct for Modding and Mapping", but translation did not reflect this.

Reply:
ok